superset/assets/src/dashboard/util/layoutHelper.js

```diff
@@ -0,0 +1,42 @@
+import {
```

Comment:
can you separate this into one function per file? "util files" can easily become unreadable and way too long. I'd prefer more files over large ones.